content/02-types.md

```diff
@@ -366,11 +366,11 @@ If the commented line is uncommented, the program does not compile because `Red`
 <!--label:types-enum-using-->
 #### Using enums
 
-Enums are a good choice if only a finite set of values should be allowed. The individual [constructors](types-enum-constructor) then represent the allowed variants and enable the compiler to check if all possible values are respected. This can be seen here:
+Enums are a good choice if only a finite set of values should be allowed. The individual [constructors](types-enum-constructor) then represent the allowed variants and enable the compiler to check if all possible values are respected:
 
 [code asset](assets/Color2.hx)
 
-After retrieving the value of `color` by assigning the return value of `getColor()` to it, a [`switch` expression](expression-switch) is used to branch depending on the value. The first three cases `Red`, `Green` and `Blue` are trivial and correspond to the constructors of `Color` that have no arguments. The final case `Rgb(r, g, b)` shows how the argument values of a constructor can be extracted: they are available as local variables within the case body expression, just as if a [`var` expression](expression-var) had been used.
+After retrieving the value of `color` by assigning the return value of `getColor()` to it, a [`switch` expression](expression-switch) is used to branch depending on the value. The first three cases, `Red`, `Green` and `Blue`, are trivial and correspond to the constructors of `Color` that have no arguments. The final case, `Rgb(r, g, b)`, shows how the argument values of a constructor can be extracted; they are available as local variables within the case body expression, just as if a [`var` expression](expression-var) had been used.
```

Comment:
`Red, Green, and Blue` (add comma?)

Reply:
TIL you can put a comma there... :) https://english.stackexchange.com/a/30519/372082

Reply:
I think in the manual I tried to be very consistent about the comma in lists of three or more items.

Reply:
This is fine with me. Do you want all lists going forward to look like "1, 2, and 3"?

Reply:
I would personally prefer it, but it is a very minor issue.

Reply:
Deciding on small things like that is the entire point :smile: it shall be done

Reply:
My personal preference is to not have a comma there. So let's go with that!

Reply:
We should list things like that in some sort of style guide somewhere. Maybe a wiki page here would be good.